pennylane/ops/qutrit/matrix_ops.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the discrete-variable quantum operations that
+accept a hermitian or an unitary matrix as a parameter.
+"""
+# pylint:disable=abstract-method,arguments-differ,protected-access
+import warnings
+import numpy as np  # pylint: disable=unused-import
```

Comment:
Why is NumPy being imported if it is not being used, or will it eventually come into play?

Reply:
It was being used by ControlledQutritUnitary before I removed it from this PR. Thus, it will get used eventually, but for now I can remove it from the PR.